src/components/NavSection.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <div class="Home">
+    <sidebar-section v-if="showSidebar" />
+    <nav>
+      <div class="logo">
+        <img src="@/assets/images/Icons/Group 2.svg" alt="" id="logo" />
+      </div>
+      <div class="nav-options">
+        <div class="nav-icon">
+          <img src="@/assets/images/Icons/Rectangle.svg" alt="" id="nav-icon" />
+        </div>
+        <ul>
+          <li> <router-link to ="/"><span>00</span>HOME</router-link></li>
```

Comment:
Create a navs property and loop through it

for example

`navs: [
  { name: 'Home', path: '/', figure: '00'},
  { name: 'Destination', path: '/destination', figure: '01'}
]
`

` <li v-for="nav in navs"> <router-link :to="nav.path"><span>{{nav.figure}}</span>{{nav.name}}</router-link></li>`